ddtrace/contrib/pytest/plugin.py

```diff
@@ -159,10 +254,16 @@ def pytest_runtest_protocol(item, nextitem):
 
         yield
 
+    test_suite_span = _extract_span(item.parent)
+    if test_suite_span is not None:
+        if nextitem is None or nextitem.parent != item.parent and not test_suite_span.finished:
+            test_suite_span.finish()
```

Comment:
this means that if it's the last test in the suite, we also finish the suite, right?

Reply:
Yes, exactly!